### ISSUE TITLE
Encoding the position in the URI doesn't seem to work.

### ISSUE THREAD

**Issue by AlyceOsbourne** [nguyenvanduocit/obsidian-open-gate#67]
When opening gates vie the URI, it seems to always use the right hand sidebar.
I have included in the uri:
- `&position=center`
- `&position="center"`
- `&position='center'`
And thus far, I have had no luck. Maybe its something I am doing wrong.
Or maybe I misunderstood the documentation

**Comment by nguyenvanduocit** [comment]
Sorry for late reply, can i confirm that the URI is not preconfigured?

```
Once again, to simplify the reuse of gates configured in the settings, the plugin automatically matches the title or URL with existing gates. If a match is found, the options from the gate configured in the settings are merged with the options specified in the note.
```

**Comment by nguyenvanduocit** [comment]
The issue is finnaly fix in 1.11.8
